easybuild/easyconfigs/r/RASPA2/RASPA2-2.0.41-foss-2020b.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'RASPA2'
+version = '2.0.41'
+
+homepage = 'https://iraspa.org/raspa/'
+description = """RASPA is a software package for simulating adsorption and diffusion of
+molecules in flexible nanoporous materials."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/iRASPA/RASPA2/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['85d99ad63b83541835f92f09dd41a3c6193a2e4d6ad45baee58ecf3b85713394']
+
+dependencies = [
+    ('OpenBLAS', '0.3.12'),
```

Comment:
This is included with the `foss` toolchain, so no need to add as a dependency, same goes for `FFTW` and `ScaLAPACK` (so no `dependencies` needed at all)

Reply:
i thought i should always state dependencies explicitly :-) removed :)